public/config.json

```diff
@@ -40,6 +40,11 @@
       "contentId": "xQaFzykq",
       "type": "playlist",
       "filterTags": "Beginner,Advanced"
+    },
+    {
+      "label": "Live",
+      "contentId": "WppSIFKF",
+      "type": "playlist"
```

Comment:
I remember we discussed that it could be useful to define it as a 'live' type. @dbudzins can we support it at the back-end side? Right now we have `playlist`, `favorites` and `continue_watching` types.

Reply:
If that's the case, we don't need the playlist routing mechanism but use a dedicated URL instead. Although, we also decided to follow the upcoming layout mechanics using the `contentType` property.

Reply:
The thing is that we may want to add new live playlists on our app config page. In this case it won't be possible to do it without additional configuration (custom param) on the playlist page. And it may be also confusing when in some cases we set the type in the config, and in other cases as a custom param

Reply:
@marcovandeveen @dbudzins what do you think?

Reply:
We also need to add custom params to the playlist for the epg source, right? My way of thinking about it is that a 'live' playlist can only be shown on a PlaylistLiveChannel page, so as part of setting up the playlist, we should make sure that it has the right params in custom params, then there's nothing to adjust in the config itself. It probably ties into the meta data management project that the OTT team is looking to tackle later this year.